notification/templates/acceptence-email-template.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <meta name="HandheldFriendly" content="true" />
+</head>
+
+<table>
+    <tbody>
+        <tr class="review-application-div">
+            <p>Dear HR Manager</p>
```

Comment:
Let's keep only one sentence in the mail. Remove other things

`{{ .AccepterEmail }} has accepted your invitation to join our unity System.`